test-common/src/main/kotlin/SøknadsbehandlingTestData.kt

```diff
@@ -557,3 +565,240 @@ fun søknadsbehandlingLukket(
+            fnr = sak.fnr,
+            fritekstTilBrev = "",
+            stønadsperiode = stønadsperiode,
+            grunnlagsdata = Grunnlagsdata.IkkeVurdert,
+            vilkårsvurderinger = when (sak.type) {
+                Sakstype.ALDER -> {
+                    Vilkårsvurderinger.Søknadsbehandling.Alder.ikkeVurdert()
+                }
+                Sakstype.UFØRE -> {
+                    Vilkårsvurderinger.Søknadsbehandling.Uføre.ikkeVurdert()
+                }
+            },
+            attesteringer = Attesteringshistorikk.empty(),
+            avkorting = AvkortingVedSøknadsbehandling.Uhåndtert.IngenUtestående.kanIkke(),
+            sakstype = sak.type,
+        )
+
+        // legg til for å oppdater stønadsperiode
+        val førOppdatertStønadsperiode = sak.copy(
+            søknader = sak.søknader.filterNot { it.id == søknad.id } + søknad, // replace hvis søknaden allerede er lagt til (f.eks hvis man først oppretter bare sak + søknad)
+            søknadsbehandlinger = sak.søknadsbehandlinger + søknadsbehandling,
+        )
+
+        // oppdater stønadsperiode via sak for å sjekke gyldigheten
+        val etterOppdatertStønadsperiode = førOppdatertStønadsperiode.oppdaterStønadsperiodeForSøknadsbehandling(
+            søknadsbehandlingId = søknadsbehandling.id,
+            stønadsperiode = stønadsperiode,
+            clock = clock,
+            formuegrenserFactory = formuegrenserFactoryTestPåDato(LocalDate.now(clock)),
+        ).getOrFail()
+
+        sak.copy(
+            søknader = sak.søknader.filterNot { it.id == søknad.id } + søknad, // replace hvis søknaden allerede er lagt til (f.eks hvis man først oppretter bare sak + søknad)
+            søknadsbehandlinger = sak.søknadsbehandlinger.filterNot { it.id == etterOppdatertStønadsperiode.id } + etterOppdatertStønadsperiode,
+        ) to etterOppdatertStønadsperiode
+    }
+}
+
+fun vilkårsvurdertSøknadsbehandlingAlder(
+    clock: Clock = fixedClock,
+    stønadsperiode: Stønadsperiode = stønadsperiode2021,
+    sakOgSøknad: Pair<Sak, Søknad.Journalført.MedOppgave> = nySakAlder(
+        clock = clock,
+    ),
+    customGrunnlag: List<Grunnlag> = emptyList(),
+    customVilkår: List<Vilkår> = emptyList(),
+): Pair<Sak, Søknadsbehandling.Vilkårsvurdert> {
+    return vilkårsvurdertSøknadsbehandling(
+        clock = clock,
+        stønadsperiode = stønadsperiode,
+        sakOgSøknad = sakOgSøknad,
+        customGrunnlag = customGrunnlag,
+        customVilkår = customVilkår,
+    )
+}
+
+fun vilkårsvurdertSøknadsbehandlingUføre(
+    clock: Clock = fixedClock,
+    stønadsperiode: Stønadsperiode = stønadsperiode2021,
+    sakOgSøknad: Pair<Sak, Søknad.Journalført.MedOppgave> = nySakUføre(
+        clock = clock,
+    ),
+    customGrunnlag: List<Grunnlag> = emptyList(),
+    customVilkår: List<Vilkår> = emptyList(),
+): Pair<Sak, Søknadsbehandling.Vilkårsvurdert> {
+    return vilkårsvurdertSøknadsbehandling(
```

Comment:
kan sikkerte asserte på sakstype på disse og for å forsikre at testene ikke bruker disse feil

Reply:
Tenkte egentlig bare jeg skulle stole på at domenet sier i fra hvis noe ikke er lov 🤷